### DISCUSSION TITLE
Proposal: Add `CITATION.cff` file to enable GitHub citation feature

### DISCUSSION THREAD

**Discussion by drupol** [typst/typst#4108] [Q&A] [answered]
Hello,

I frequently need to cite Typst in the documents I write, and noticed that GitHub supports a specific citation file format (`CITATION.cff`). Having this file would enhance the visibility and proper citation of Typst directly from the GitHub repository page.

GitHub's support for the `CITATION.cff` files enables easy access to citation information, which can be automatically parsed and displayed on the project page. More details can be found in the [GitHub documentation on citation files](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files).

Would you be interested in adding such a file to the project?

~~Additionally, I attempted to access Laurenz's thesis via the link https://www.user.tu-berlin.de/laurmaedje/programmable-markup-language-for-typesetting.pdf, but it appears to be unavailable. Could you please confirm if there is an updated link or alternative source for this document?~~ Eventually, a DOI ?

Thanks!

**Comment by laurmaedje** [comment, marked as answer]
I had to reupload my thesis because my TU Berlin account was deprovisioned. There is a new stable link at https://laurmaedje.github.io/programmable-markup-language-for-typesetting.pdf

**Comment by drupol** [comment]
The PR https://github.com/typst/typst/pull/4109 has been created, closing the discussion.
